### ISSUE TITLE
Seller Object Query

### ISSUE THREAD

**Issue by brahmac** [akaszynski/keepa#83]
Can I query the Keepa Seller Object using this python package? If so, what is the format for the query? I don't see any documentation for this.
Thanks.

**Comment by callum80041** [comment]
Also looking at this to use a request for a list of ASINS for a select Seller ID - any plans to update the Docs?

**Comment by akaszynski** [comment]
Sorry, been busy with my day job.  This being an open-source project, it's sometimes tough to find time to maintain this as it's now orthogonal to my current work and I'm not being paid to support his at the moment.

**Comment by Ryan-Daly** [comment]
Docs are available for this at https://keepaapi.readthedocs.io/en/latest/api_methods.html, search for seller_query.

However, the force refresh for the method doesn't work. 

example:
api.seller_query('A12G...', 'GB', storefront=True, update=0)

This isn't running an update on the storefront, it's just showing what keepa has stored in it database

**Comment by akaszynski** [comment]
Could you try `update=30`?

**Comment by Ryan-Daly** [comment]
I don't think update works how its supposed too so I've contacted keepa. It's not this library.

I've been testing it and seem to get the update as slow as an hour later. I kept making the same request until i got updated results. Then the 'last updated' time said the time of my first request. So definitely a problem with keepa, not with this library.

Also, I tried with a direct get request without this library and it was the same.
